lib/gui/modules/progress-status.js

```diff
@@ -41,7 +41,7 @@ exports.fromFlashState = (state) => {
   const shouldValidate = settings.get('validateWriteOnSuccess')
   const shouldUnmount = settings.get('unmountOnSuccess')
 
-  if (state.percentage === utils.PERCENTAGE_MINIMUM && state.speed == null) {
+  if (state.percentage === utils.PERCENTAGE_MINIMUM && state.speed === null) {
```

Comment:
Is `state.speed` going to be `null` , or could it also be undefined?

Reply:
It should never be `undefined`, but its default state is `null` as `0` doesn't necessarily indicate that progress isn't happening as you highlighted